### PR TITLE
Update release 20200930 Run update-apimanager script to upgrade API Manager

### PR DESCRIPTION
**Pull request by mvenzke-axway**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by mvenzke-axway** [comment]
This was worded as though the requirement to do an update had been removed, when only a certain script had been removed.  I made it clear that you need to follow that link and read the instructions for how updating works now because I've seen people missing the need to update.

File: content/en/docs/apim_relnotes/20200930_apimgr_relnotes.md [hunk 1]
```diff
@@ -229,7 +229,7 @@ To stay current and align our offerings with customer demand and best practices,
 
 ### Run `update-apimanager.py` script to upgrade API Manager
 
-The requirement to run `update-apimanager.py` has been removed. For more information see [Update API Manager](/docs/apim_installation/apigw_upgrade/upgrade_steps_oneversion/#update-api-manager).
+The script `update-apimanager.py` has been removed, but **you still need to perform an update of the API Manager FED file**. For instructions on how, see [Update API Manager](/docs/apim_installation/apigw_upgrade/upgrade_steps_oneversion/#update-api-manager).
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20200930_apimgr_relnotes.md:232]: @maherb4 @cpalmeraxway please review this

**Review by maherb4 — APPROVED** [review]
Good clarification on the upgrade process

**Review by andreamussap — APPROVED** [review]
https://deploy-preview-1417--axway-open-docs.netlify.app/docs/apim_relnotes/20200930_apimgr_relnotes/#removed-features

**Comment by andreamussap** [comment]
@mvenzke-axway Thank you for your contribution. Just 2 tips to make your PRs more meaningful :)

- You can edit the title of the PR to make it more descriptive.
- You can edit the description of the PR to tell what it is about, instead of adding a new comment

For more information, see https://axway.jiveon.com/groups/api-mgt/blog/2020/10/12/axway-open-docs-contributing-via-github

**Pull request merged**